File: src/main/java/one/oktw/sponge/internal/CommandRegister.java
package one.oktw.sponge.internal;

import one.oktw.sponge.Main;
import one.oktw.sponge.command.*;
import org.spongepowered.api.Sponge;
import org.spongepowered.api.command.CommandManager;

public class CommandRegister {

    public CommandRegister() {
        Main plugin = Main.getMain();
        CommandManager commandManager = Sponge.getCommandManager();

        commandManager.register(plugin, new CommandCreate().getSpec(), "create");
        commandManager.register(plugin, new CommandWorld().getSpec(), "world", "w");
        commandManager.register(plugin, new CommandRebuild().getSpec(), "rebuild");
        commandManager.register(plugin, new CommandHat().getSpec(), "hat");
        commandManager.register(plugin, new CommandSpawn().getSpec(), "spawn");
        commandManager.register(plugin, new CommandInvite().getSpec(), "invite", "inv", "i");
        commandManager.register(plugin, new CommandRemove().getSpec(), "remove", "rm");
        commandManager.register(plugin, new CommandHub().getSpec(), "hub", "h");
    }
}

File: src/main/kotlin/one/oktw/galaxy/internal/register/CommandRegister.kt
package one.oktw.galaxy.internal.register

import one.oktw.galaxy.Main.Companion.main
import one.oktw.galaxy.command.Admin
import one.oktw.galaxy.command.Sign
import one.oktw.galaxy.command.Spawn
import org.spongepowered.api.Sponge

class CommandRegister {
    private val logger = main.logger

    init {
        logger.info("Register command...")
        Sponge.getCommandManager().apply {
            register(main, Sign().spec, "sign")
            register(main, Admin().spec, "admin")
            register(main, Spawn().spec, "spawn")
        }
    }
}
